packages/edit-site/src/components/navigation-sidebar/navigation-panel/index.js

```diff
@@ -82,23 +82,24 @@ const NavigationPanel = ( { isOpen, setIsOpen, activeTemplateType } ) => {
 							<NavigationGroup title={ __( 'Editor' ) }>
 								<NavigationItem
 									title={ __( 'Site' ) }
-									href={ addQueryArgs( '', {
-										page: 'gutenberg-edit-site',
+									href={ addQueryArgs( window.location.href, {
+										postId: undefined,
```

Comment:
Having to reset `postId` and `postType` has probably the same amount of efforts as having to pass `page=gutenberg-edit-site` 😅 . In the future, we can hopefully abstract this away in #36488 so I don't think it matters either way.

Reply:
Yeah a router would be nice 😀